Pages/Blog.php

```diff
@@ -84,6 +84,24 @@ public function get() {
             }
         }
 
+        foreach($blog->posts as $key => $post) {
+            //Header images
+            $header_image = '';
+
+            if(empty($post['header_image']) OR $post['header_image'] == '') {
```

Comment:
this is redundant - if $post['header_image'] == '' it will return true for empty()